author/vpc.tf

```diff
@@ -0,0 +1,71 @@
+# Create a VPC to launch our instances into
+resource "aws_vpc" "author-vpc" {
+  cidr_block = "${var.vpc_ip_block}"
+  enable_dns_support = true
+  tags {
+    Name = "${var.env}-author-vpc"
+  }
+}
+
+# Create an internet gateway to give our subnet access to the outside world
+resource "aws_internet_gateway" "author-gateway" {
+  vpc_id = "${aws_vpc.author-vpc.id}"
+  tags {
+    Name = "${var.env}-author-internet-gateway"
+  }
+}
+
+# Grant the VPC internet access on its main route table
+resource "aws_route" "internet_access" {
+  route_table_id         = "${aws_vpc.author-vpc.main_route_table_id}"
+  destination_cidr_block = "0.0.0.0/0"
+  gateway_id             = "${aws_internet_gateway.author-gateway.id}"
+}
+
+resource "aws_subnet" "author_application" {
+  vpc_id                  = "${aws_vpc.author-vpc.id}"
+  cidr_block              = "${var.application_cidr}"
+  availability_zone       = "eu-west-1a"
+  tags {
+    Name = "${var.env}-author-application-subnet"
+  }
+}
+
+# Create a subnet to launch our database into.
+resource "aws_subnet" "database-1" {
```

Comment:
Might be helpful to rename these to `database-1c` and `database-1b` so we can easily grok what the subnets map to, as the only reason we have two is for the cross availability zone support.